core/dbt/task/printer.py

```diff
@@ -144,9 +157,11 @@ def print_run_end_messages(results, keyboard_interrupt: bool = False) -> None:
     )
 
     for error in errors:
-        print_run_result_error(error, is_warning=False)
+        group = groups.get(error.node.unique_id) if groups and hasattr(error, "node") else None
```

Comment:
This is neat, iiuc this is printing the summary of all model errors with the group info for those models

Reply:
Do these summary logs have a different code (like how individual model failures have "Z002")?

Reply:
Yes:
1. `RunResultWarning` – Z021
2. `RunResultFailure` – Z022
3. `RunResultError` – Z024